examples/parquet_analytics_demo.ipynb

```diff
@@ -0,0 +1,527 @@
+    "\n",
+    "* `arrow`\n",
+    "* `ggplot2`\n",
+    "* `jsonlite`\n",
+    "* `geojsonio`\n",
+    "* `sf`\n",
+    "* `tidyverse`\n",
+    "* `AzureStor`\n",
+    "* `AzureRMR`\n",
+    "\n",
+    "which you can install by running `install.packages(c(\"AzureStor\", \"AzureRMR\", \"arrow\", \"ggplot2\", \"jsonlite\", \"geojsonio\", \"sf\", \"tidyverse\")`. These packages allow us to interact with Microsoft Azure, load parquet data, interact with and format data the way we want, and visualize our result. If you're using a different cloud platform for file storage and management, you may need other packages instead of these, but for our demo, we'll use these two to handle connecting with Azure.\n",
+    "\n",
+    "While common practice is to use the `rgdal` package for spatial analytics purposes (see last example), `rgdal` will reach its end-of-supported lifecycle in early 2023. Hence, we recommend the new industry standard `sf` package (standing for \"simple features\")."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f2e39b1c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import relevant packages\n",
+    "library(arrow)\n",
+    "library(jsonlite)\n",
+    "library(AzureStor)\n",
+    "library(AzureRMR)\n",
+    "library(sf)\n",
+    "library(geojsonio)\n",
+    "library(tidyverse)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "acdcd7b9",
+   "metadata": {},
+   "source": [
+    "## Configuring variables and our notebook environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9a58077",
+   "metadata": {},
+   "source": [
+    "Before we get into the code proper, we need to configure a few variables so that things run smoothly. There are four variables that correspond to accessing a cloud-hosted platform directly through R. They will, respectively, allow us to make an authorized connection, to a particular storage provider, so that we can access the endpoint where the files live, within a specific storage container. For this particular implementation, we're hosting files in an Azure Gen. 2 storage bucket, but the principles are the same regardless of the hosting platform."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "43223098",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "TENANT_ID = \"YOUR TENANT ID STRING\"\n",
+    "HOST = \"URL TO YOUR STORAGE PROVIDER\" # for example, https://storage.azure.com\n",
+    "ADLS_ENDPOINT = \"YOUR SPECIFIC ENDPOINT\"\n",
+    "CONTAINER = \"NAME OF YOUR STORAGE CONTAINER\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d50161e9",
+   "metadata": {},
+   "source": [
+    "For ease of visualization, we'll also modify the default graphics window that R uses within jupyter notebook, so that our graphs are easier to see."
```

Comment:
Would it be worth pointing out to users that they can change the values based on their preferences?